### ISSUE TITLE
Background color does not follow dark or light theme

### ISSUE THREAD

**Issue by chitgoks** [akshathjain/sliding_up_panel#144]
Background is always white.

**Comment by essboyer** [comment]
Same issue here. I got around it by wrapping the content (a Column in my case) with a Container, and setting the background color to `Theme.of(context).backgroundColor`.

**Comment by akshathjain** [comment]
@essboyer that works but it might be easier to use the built in color property instead of wrapping the content.

**Comment by chitgoks** [comment]
yes. that was my workaround but not everybody may know about it right away especially newbies.

**Comment by akshathjain** [comment]
Will add default value to the color property to reflect the app theme
